### DISCUSSION TITLE
Is there is a way to override global defaults for delays/closeDelays for Hover Cards, Tooltips etc.?

### DISCUSSION THREAD

**Discussion by RIP21** [chakra-ui/ark#2587] [Ideas]
I want to override defaults as defaults are too slow for us. I assume I can hack around it using `defaultProps` (I think they will be deprecated AFAIR?), but having some sort of global Provider of some sort to override these would be a very nice feature.

**Comment by segunadebayo** [comment]
There's currently no way to do this. Feel free to implement it in your userland code and contribute upstream.

I imagine this could be a global context for `delays`

**Reply by RIP21** [reply]
I imagine that being tiny wrappers for each `Root` component with delay functionality that would inject these globally from context if so (context) is present.
Makes sense. Even tho knowing that wrappers aren't that big of a deal in terms of performance (like not a big deal at all as it's just a fn call mostly), I am still trying to avoid them for some reason :D

Will think about contributing something to Ark tho as that feels like the right place for that thing.

**Comment by segunadebayo** [comment]
Closing — global defaults are not planned. Delays can be configured per-instance via props.
